game/models.py

```diff
@@ -0,0 +1,56 @@
+from settings import PLAYER_LIVES, ALLOWED_ATTACKS
+from game.exceptions import GameOver, EnemyDown
+from game.validation import is_valid_attack
+import random
+
+
+# Player class to represent a player in the game
+class Player:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.lives = PLAYER_LIVES
+        self.score = 0
+
+    # Method to select an attack from the user
+    def select_attack(self) -> str:
+        while True:
+            attack = input("Choose your attack (1 - Paper, 2 - Stone, 3 - Scissors): ")
+            if is_valid_attack(attack):
+                return ALLOWED_ATTACKS[attack]
+            else:
+                print("Invalid attack. Please select 1, 2, or 3.")
+
+    # Method to decrease the number of lives
+    def decrease_lives(self) -> None:
+        self.lives -= 1
+        if self.lives == 0:
```

Comment:
Actually, you're interested in the scenario when `lives` value goes under `1`.
Equality comparison to `0` makes the behavior unclear.
```suggestion
        if self.lives < 1:
```